requirements.txt

```diff
@@ -7,3 +7,4 @@ pyminifier
 funcparserlib
 iso8601
 colorama
+jsonschema
```

Comment:
What is using this?

Reply:
the path in my IDE must be pulling in the wrong things.  What was using jsonschema was /user-scratch/hiderrt1/scrape_boss/validator.py.  I need to track down why it was loading this validator.py in the first place.

Reply:
Removed the /user-scratch directories that were listed as sources in my IDE and it worked without jsonschema